source/release_notes.rst

```diff
@@ -6,6 +6,71 @@ Release Notes
 .. contents:: :local:
 
 
+--------
+
+ww201840
+========
+
+================================  =====
+Build ID                          
+Software Version                  Android version 9
+Mesa                              18.2.0-rc4
+Kernel Version                    4.14.67
+Download Link                     https://github.com/projectceladon/celadon-binary/tree/master/ww201840
+================================  =====
+
+Integrated Features
+-------------------
+
+* Graphics Memory Allocator (Gralloc) 1.0
+* HDMI display support
+* HWC 2.0 support for Display
+* USB digital audio playback support
+* Analog Audio (MP3/AAC) playback on stereo wired Headset
+* Audio parameter framework for audio routing policy
+* Wi-Fi 802.11 a/b/g/n
+* Wired Ethernet support
+* Android Grub boot support
+* Android Kernel Flinger boot support
+* Thermal Daemon is enabled for CELADON
+* The HW media codec is enabled for CELADON.
+	HW codec capabilities supported : 
+		Decoder: h264, h265, vp8, vp9, jpeg 
+		Encoder: h264, h265(WIP, will ready in early WW32), jpeg
+
+• Media sdk 4K support is enabled on CELADON
```

Comment:
Start the line with '*' to be consistent with the previous bullets. Same for the bullet below.